R/alexanderZTest.R

```diff
@@ -0,0 +1,166 @@
+.z.test <- function(x, y=NULL, alternative="two.sided", mu=0, sigma.x=1, sigma.y=1, conf.level=0.95, 
+                    paired=FALSE, ciValueMeanDiffES=0.95) {
+  if (!is.null(y)) 
+    stop("Paired and two-sample z.test not yet implemented")
+  
+  result <- list(statistic=NULL, p.value=NULL, conf.int=NULL, estimate=NULL, null.value=mu, 
+                 alternative=alternative, method=NULL, data.name=NULL, 
+                 d=NULL, confIntEffSize=NULL)
+  
+  nX <- length(x)
+  nY <- NULL
+  
+  if (nX <= 1) 
+    stop("not enough x observations")
+  
+  xBar <- mean(x)
+  
+  if (is.null(y)) {
+    dname <- deparse(substitute(x))
+    
+    method <- "One-sample z-Test"
+    estimate <- xBar - mu
+    names(estimate) <- "mean difference of x"
```

Comment:
if these strings are ever shown to users, they need a `gettext`. If not, this is fine.

Reply:
This is totally irrelevant in the JASP setting, but I just added so it'll function independently as an R function. I think it doesn't hurt to keep it that way. Unless you think otherwise.